### DISCUSSION TITLE
[Feature request] History sidebar grouped by date

### DISCUSSION THREAD

**Discussion by alfredo-f** [danny-avila/LibreChat#192] [Feature Requests & Suggestions]
ChatGPT recently introduced the history sidebar being grouped by date

![image](https://user-images.githubusercontent.com/22865959/235244257-a14fd818-671b-4491-b8b5-b0977a625926.png)

Any idea if it's possible to implement this?

**Comment by danny-avila** [comment]
Yes I'll put this on the to-do! Wow, i was on the site earlier today and I didn't see this
